scripts/cibuild/PublishArtifactsFromVsts.ps1

```diff
@@ -67,7 +67,7 @@ if(Test-Path $NupkgsDir)
 {
     Write-Output "Publishing nupkgs from '$Nupkgs'"
     # Push all nupkgs to the nuget-build feed on myget.
-    Get-Item "$NupkgsDir\*.nupkg"  | Push-ToMyGet -ApiKey $NuGetBuildFeedApiKey -BuildFeed $NuGetBuildFeedUrl -SymbolSource $NuGetBuildSymbolsFeedUrl
+    Get-Item "$NupkgsDir\*.nupkg" -Exclude "Test.Utility.*.nupkg" | Push-ToMyGet -ApiKey $NuGetBuildFeedApiKey -BuildFeed $NuGetBuildFeedUrl -SymbolSource $NuGetBuildSymbolsFeedUrl
```

Comment:
Thoughts on making the exclude `*Test*.nupkg` to be more future proof?

Reply:
i'd say do Test*

Reply:
I was thinking that maybe sometime `Test.Utility` would be renamed to `NuGet.Test.Utility`.

Reply:
Fixed.